frontend/src/components/dashboard/data/Demo.vue

```diff
@@ -172,76 +180,73 @@
 		<label for="majorRadio" class="ml-2">College Major</label>
 	    </div>
 	</div>
-
-	<div class="card flex justify-content-center">
-            <MultiSelect v-model=genderQuery :options=genderOptions filter optionLabel="label" optionValue="value" placeholder="Gender" class="w-full md:w-20rem" @update:modelValue=updateGender />
-
-	    <MultiSelect v-model=ageQuery :options=ageOptions filter optionLabel="label" optionValue="value" placeholder="Age Group" class="w-full md:w-20rem" @update:modelValue=updateAge />
-
-	    <MultiSelect v-model=majorQuery :options=majorOptions filter optionLabel="label" optionValue="value" placeholder="College Major" class="w-full md:w-20rem" @update:modelValue=updateMajor />
-
-	</div>
-	<div>
-	    <Button v-if="tableItems.length > 0" label="Download CSV" @click="download('results', tableItems)" :loading="loading_download" />
-	</div>
-
-	<div class="filters">
-	    <div class="filter-text">Filters</div>
-	    <FilterChip :options="filter.options" v-for="filter in filters">{{ filter.name }}</FilterChip>
+    </div>
+    <div class="filters">
+	<div class="filter-text">Filters</div>
+	<FilterChip @update-filter=updateFilter :id="filter.id" :options="filter.options" v-for="filter in filters" :key="compare">{{ filter.name }}</FilterChip>
```

Comment:
The key should be unique for each item in the v-for. Using something like the filter.id would be appropriate here. See https://vuejs.org/guide/essentials/list.html#maintaining-state-with-key

Reply:
_fixed_